src/vellum/workflows/utils/functions.py

```diff
@@ -0,0 +1,13 @@
+from typing import Callable
+
+from vellum.client.types.function_definition import FunctionDefinition
+
+
+def compile_function_definition(function: Callable) -> FunctionDefinition:
```

Comment:
I debated whether I should start the `src/vellum/prompts` module and put this in `src/vellum/prompts/utils/functions.py` instead. Thoughts?

Reply:
I'm game!